### PR TITLE
Adding dfmt support for D

### PR DESCRIPTION
**Pull request by LaurentTreguier**
This PR adds basic dfmt support.

### CODE REVIEW

**Comment by Glavin001** [comment]
Awesome! Please add tests as well. See step 4 of `How to add a beautifier for a language`: https://github.com/Glavin001/atom-beautify/blob/master/docs/add-languages-and-beautifiers.md#how-to-add-a-beautifier-for-a-language

**Comment by LaurentTreguier** [comment]
I added a simple test and put dfmt as default beautifier, since it seems better than uncrustify for beautifying D (with the code I tested them with at least), and also because the test would fail otherwise.

File: src/beautifiers/dfmt.coffee [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Requires [dfmt](https://github.com/Hackerpilot/dfmt)
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Dfmt extends Beautifier
+  name: "dfmt"
+
+  options: {
+    D: true
```
Comment (Glavin001) [inline src/beautifiers/dfmt.coffee]: Should be `false`

**Comment by Glavin001** [comment]
Published to v0.28.23

**Pull request merged**